.prettierignore

```diff
@@ -6,3 +6,6 @@ pnpm-lock.yaml
 /app/src/lang/translations/*.yaml
 !/app/src/lang/translations/en-US.yaml
 /docs/packages/
+storybook-static/
+uploads/
+extensions/
```

Comment:
Can we use more specific paths for these like in the `.gitignore` file?

https://github.com/directus/directus/blob/675ec625e1854aa5b57c4f421a26d89f024dc052/.gitignore#L47-L48